### PR TITLE
fix(obs-backgroundremoval-git): fix build

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

File: packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript [hunk 1]
```diff
@@ -15,19 +15,21 @@ pkgver() {
 version="$(pkgver)"
 incompatible=('ubuntu:jammy' 'ubuntu:focal' 'ubuntu:bionic' 'debian:buster' 'debian:bullseye')
 
+prepare() {
+  cd vendor
+  git clone https://github.com/curl/curl.git --depth=1
+}
```
Comment (Elsie19) [inline packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript:21]: Why are we cloning curl?

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Fix title

File: packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript (hunk 1 above)
Comment (Zahrun) [reply packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript:21]: Not needed

**Comment by Zahrun** [comment]
> Fix title

As in it's a fix?

**Comment by Elsie19** [comment]
> > Fix title
> 
> As in it's a fix?

add the fix description: <kbd>fix(obs-backgroundremoval-git): description</kbd>

**Comment by Elsie19** [comment]
> mv: cannot move '/usr/src/pacstall/obs-backgroundremoval-git/usr/lib/x86_64-linux-gnu' to a subdirectory of itself, '/usr/src/pacstall/obs-backgroundremoval-git/usr/lib/x86_64-linux-gnu/x86_64-linux-gnu'

File: packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript [hunk 2]
```diff
@@ -16,18 +16,15 @@ version="$(pkgver)"
 incompatible=('ubuntu:jammy' 'ubuntu:focal' 'ubuntu:bionic' 'debian:buster' 'debian:bullseye')
 
 build() {
-  mkdir build && cd build || exit
-  cmake -DUSE_SYSTEM_ONNXRUNTIME=ON -DCMAKE_INSTALL_PREFIX:PATH="${pkgdir}"/usr .. > /dev/null && cmake --build . -j"$(nproc)" > /dev/null
+  mkdir build_x86_64 && cd build_x86_64 || exit
```
Comment (Elsie19) [inline packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript:19]:
```suggestion
  mkdir build_x86_64 && cd build_x86_64 || return 1
```

File: packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript [hunk 3]
```diff
@@ -16,18 +16,15 @@ version="$(pkgver)"
...
+  cmake --preset linux-x86_64 -DCMAKE_INSTALL_PREFIX:PATH="${pkgdir}"/usr .. > /dev/null
+  cmake --build . -j"$(nproc)" > /dev/null
 }
 
 package() {
-  cd build || exit
+  cd build_x86_64 || exit
```
Comment (Elsie19) [inline packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript:25]:
```suggestion
  cd build_x86_64 || return 1
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript [hunk 4]
```diff
@@ -16,18 +16,15 @@ version="$(pkgver)"
 incompatible=('ubuntu:jammy' 'ubuntu:focal' 'ubuntu:bionic' 'debian:buster' 'debian:bullseye')
 
 build() {
-  mkdir build && cd build || exit
-  cmake -DUSE_SYSTEM_ONNXRUNTIME=ON -DCMAKE_INSTALL_PREFIX:PATH="${pkgdir}"/usr .. > /dev/null && cmake --build . -j"$(nproc)" > /dev/null
+  mkdir build_x86_64 && cd build_x86_64 || return 1
+  cmake --preset linux-x86_64 -DCMAKE_INSTALL_PREFIX:PATH="${pkgdir}"/usr .. > /dev/null
+  cmake --build . -j"$(nproc)" > /dev/null
```
Comment (Elsie19) [inline packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript:21]:
```suggestion
  cmake --build . -j"${NCPU}" > /dev/null
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**